easybuild/easyconfigs/c/Clang/Clang-16.0.6-GCCcore-12.3.0-shared.eb

```diff
@@ -0,0 +1,65 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2013-2015 Dmitri Gribenko, Ward Poelmans
+# Authors:: Dmitri Gribenko <gribozavr@gmail.com>
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+# This version is a modification of the already existing, static library build.
+# For the HOOMD-blue installation we require the dynamic libraries. Thus, flang and lldb
+# were removed as they cause problems when building.
+# Authors:: J. Sassmannshausen (Imperial College London/UK)
+##
+
+name = 'Clang'
```

Comment:
This should use the new LLVM easyblock instead once it's been merged

Reply:
Apparently not, since that easyblock only handles LLVM/CLang >= 18.1.6 for non miminal builds.